ote/resources/public/language/en.edn

```diff
@@ -688,6 +689,9 @@ You can also draw the operating area or point to the map with drawing tools. You
   :help-about-ytj-link "https://www.ytj.fi/en/index/whatisbis.html"
   :help-ytj-contact-change-link-desc "Change details of your business in the Business Information System (BIS)"
   :help-ytj-contact-change-link "https://www.ytj.fi/en/index/notifications/notificationsofchanges.html"
+  :help-contact-details "Please enter contact details, using which a developer of traffic services or other similar partner may contact your company or organization."
+  :help-contact-details-privacy "The details provided will be publicly available in NAP -transport catalogue. Please note this, if the contact details contain personal information."
```

Comment:
Not sure, but I think that there should not be comma (,) before if

Reply:
Strings will in any case updated by localization round, will update whatever comes from there :)